buscar-arreglo.rb

```diff
@@ -0,0 +1,14 @@
+def buscar(pArray, pOtro)  
```

Comment:
snake_case is the convention for methods and variables in ruby.
ex:
```suggestion
def buscar(first_array, other_array)  
```